### PR TITLE
Latest Jan/Feb updates.

### PR DESCRIPTION
**Pull request by givenscj**
Updates for new Sentinel screens, addresses latest issues.

### CODE REVIEW

File: Hands-on lab/Before the HOL - Security baseline on Azure.md [hunk 1]
```diff
@@ -129,16 +118,44 @@ Synopsis: In this exercise, you will set up your environment for use in the rest
...
+
+1. Open a browser window to the cloud workshop GitHub repository (<https://github.com/microsoft/MCW-Security-baseline-on-Azure>).
+
+2. Select **Clone or download**, then select **Download Zip**.
+
+    ![Clone or download and Download ZIP are highlighted in this screenshot of the cloud workshop GitHub repository.](images/Hands-onlabstep-bystep-Azuresecurityprivacyandcomplianceimages/media/image3.png "Clone or Download Zip")
+
+3. Extract the zip file to your local machine, be sure to keep note of where you have extracted the files (such as `c:\temp`). You should now see a set of folders:
+
+    ![A set of extracted folders and files are visible in File Explorer: .vs, AzureTemplate, Database, Scripts, WebApp, README.md.](images/Hands-onlabstep-bystep-Azuresecurityprivacyandcomplianceimages/media/image4.png "Extract the zip file")
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/Before the HOL - Security baseline on Azure.md]: Please double check me.  The alt-text says the files visible are .vs, AzureTemplate, Database, Scripts, WebApp….etc., but the image shows different files. (Hands-on lab, Whiteboard design session, HTMLLINKS.md, License, README.md).  Thanks!

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Contoso Ltd is a multinational corporation, headquartered in the United States t
 
 They are exploring a lift and shift strategy to Azure, but have a large focus on Azure Security and Privacy features.
 
-October 2019
+February 2020
 
 ## Target audience
```
Comment (DawnmarieDesJardins) [inline README.md]: Please review Azure Services and Related Products section.  The lab requirements list Visual Studio 2019 & SQL Management Studio but neither of those are on the ReadMe page.

File: Hands-on lab/HOL step-by step - Security baseline on Azure.md [hunk 3]
```diff
@@ -214,73 +214,77 @@ Synopsis: In this exercise, attendees will utilize Azure SQL features to data ma
...
 
+    > **Note**: If prompted, login using your Azure / MSDN account.
+
 2. In the **Solution Explorer**, navigate to and double-click the **Web.config** file to open it.
 
-    ![Web.config is highlighted under InsuranceAPI in Solution Explorer.](images/Hands-onlabstep-bystep-Azuresecurityprivacyandcomplianceimages/media/image28.png "Open Web.config")
+    ![Web.config is highlighted under the InsuranceAPI project in Solution Explorer.](images/Hands-onlabstep-bystep-Azuresecurityprivacyandcomplianceimages/media/image28.png "Open Web.config")
 
-3. Update the web.config (line 77) to point to the **Insurance** database created in Task 2. You should only need to update the server name to point to your Azure SQL Server.
+3. Update the web.config (line 72) to point to the **Insurance** database created in Task 2. You should only need to update the server name to point to your Azure SQL Server.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - Security baseline on Azure.md]: Instructions said change line 77 but the alt-text and the image showed line 72.  I changed instructions to match the image.  Please verify this was correct.

File: Hands-on lab/HOL step-by step - Security baseline on Azure.md [hunk 4]
```diff
@@ -344,7 +348,7 @@ Synopsis: In this exercise, attendees will utilize Azure SQL features to data ma
 
 9. Select **Cancel**.
 
-10. Switch back to the Azure Portal, and select the User.SSN data masking.
+10. Switch back to the Azure Portal, and select the User_SSN data masking.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - Security baseline on Azure.md]: Changed this from User.SSN to User_SSN as the Edit Masking Rule image below shows dbo_User_SSN.  Please verify this is now correct.

File: Hands-on lab/HOL step-by step - Security baseline on Azure.md [hunk 5]
```diff
@@ -380,9 +388,9 @@ Synopsis: In this exercise, attendees will utilize Azure SQL features to data ma
 
 24. Select **Finish**, and the configured will start.
 
-    > **Note**: You may receive a "wrapKey" error. If so, ensure that your account has been assigned that permission in the Azure Key Vault.
+    > **Note**: You may receive a "Wrap Key" error. If so, ensure that your account has been assigned that permission in the Azure Key Vault.
```
Comment (DawnmarieDesJardins) [inline Hands-on lab/HOL step-by step - Security baseline on Azure.md]: Changed instructions from "wrapKey" to Wrap Key" as alt-text and image below show it as two words, capitalized.  Please verify this is now correct.

**Comment by DawnmarieDesJardins** [comment]
@givenscj @timahenning 
Hi Chris!  Thanks for the test/fix.  I added some changes/comments above.  Please review and advise.  Have a great day, Dawnmarie

**Comment by DawnmarieDesJardins** [comment]
@givenscj @timahenning 
Hi Chris, please review the above and advise.  Once these are resolved, I can merge and update the HTML files.  Thanks much, Dawnmarie

File: Hands-on lab/Before the HOL - Security baseline on Azure.md (hunk 1 above)
Comment (givenscj) [reply Hands-on lab/Before the HOL - Security baseline on Azure.md]: Thanks. Your change is correct.

File: README.md (hunk 2 above)
Comment (givenscj) [reply README.md]: Those aren't a focus of the MCW and are simply tools they use to work with the focus products.

File: Hands-on lab/HOL step-by step - Security baseline on Azure.md (hunk 3 above)
Comment (givenscj) [reply Hands-on lab/HOL step-by step - Security baseline on Azure.md]: It is line 77, I updated the screen shot and doc to match.

File: Hands-on lab/HOL step-by step - Security baseline on Azure.md (hunk 4 above)
Comment (givenscj) [reply Hands-on lab/HOL step-by step - Security baseline on Azure.md]: Thanks. This is ok.

File: Hands-on lab/HOL step-by step - Security baseline on Azure.md (hunk 5 above)
Comment (givenscj) [reply Hands-on lab/HOL step-by step - Security baseline on Azure.md]: Thanks, also add a extra part for the actual permission name.

**Comment by givenscj** [comment]
Ok, all checked in.

File: Hands-on lab/Before the HOL - Security baseline on Azure.md (hunk 1 above)
Comment (DawnmarieDesJardins) [reply Hands-on lab/Before the HOL - Security baseline on Azure.md]: @givenscj Sorry, I wasn't clear - I didn't change anything on this one.  I'm not sure if the alt-text or the image is incorrect.  Should we be seeing the folders mentioned in the alt-text, or is the image correct and the alt-text should be corrected?

**Pull request merged**